src/main/kotlin/fi/elsapalvelu/elsa/service/impl/TyoskentelyjaksoServiceImpl.kt

```diff
@@ -373,10 +375,6 @@ class TyoskentelyjaksoServiceImpl(
         if (tyoskentelyjaksonPituus > 0) {
             // Summataan suoritettu aika koulutustyypettäin
             if (isYek) {
-                val yliopistosairaalaMaksimi = 182.5
-                val keskussairaalaMaksimi = 182.5
```

Comment:
Mistä nämä maksimit ovat tulleet? EIkö sairaalatyöstä lasketa kaikkea? Tein korjauksia tiketin pohjalta, mutta jäin vielä miettimään tätä kohtaa

Reply:
Outoja lukuja, opinto-oppaan mukaan on vain vähimmäismääriä eri työskentelypaikoille.

Reply:
Kirsi kuittasi, että laskurin luvut näyttivät oikealta